### PR TITLE
Show notice when syncing

### PR DESCRIPTION
**Pull request by stage-rl**

### CODE REVIEW

**Comment by stage-rl** [comment]
Screenshot_20240119_103828

File: app/controllers/boxes_controller.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ def sync_all
     EventBus.publish(:box_sync_all_requested)
 
     Current.tenant.boxes.sync_all
+    redirect_back_or_to message_threads_path, notice: t("boxes.sync_all_running")
```
Comment (jsuchal) [inline app/controllers/boxes_controller.rb:16]: ked je to event `:box_sync_all_requested` tak aj tu by som cakal 
```suggestion
    redirect_back_or_to message_threads_path, notice: t("boxes.sync_all_requested")
```

File: config/locales/sk.yml [hunk 2]
```diff
@@ -244,6 +244,8 @@ sk:
   sender_uri: "URI odosielateľa"
   recipient_uri: "URI prijímateľa"
   box: "Schránka správy"
+  boxes:
+    sync_all_running: "Synchronizácia správ bola spustená a prebieha na pozadí"
```
Comment (jsuchal) [inline config/locales/sk.yml:248]: Prilis technicky slovnik na moj vkus a ked je tu taketo tlacitko, tak drzime rovnaky slovnik.

image


```suggestion
    sync_all_running: "Sťahovanie nových správ bolo spustené."
```

**Review by jsuchal — CHANGES_REQUESTED** [review]
Nejake male veci

**Review by jsuchal — APPROVED** [review]

**Pull request merged**